otelcol/collector.go

```diff
@@ -247,6 +256,31 @@ func (col *Collector) setupConfigurationComponents(ctx context.Context) error {
 	return nil
 }
 
+func (col *Collector) newReloadServer() (*http.Server, error) {
+	server := &http.Server{
+		Addr: ":8888",
+	}
+
+	http.HandleFunc("/-/reload", col.reloadHandler)
```

Comment:
Was there a consensus somewhere on the endpoint's path? I'm only seeing one issue where you're the only commenter.